crt0stack/src/lib.rs

```diff
@@ -925,4 +925,45 @@ mod tests {
         assert_eq!(reader.next(), None); // terminator
         assert_eq!(reader.next(), None); // don't overrun
     }
+
+    #[test]
+    fn crt0_stack_alignment() {
+        // Prepare the crt0 stack.
+        #[repr(C, align(32))]
+        struct Aligned<T>(T);
+
+        let mut aligned = Aligned([0u8; 1024]);
+        let mut error = false;
+        for i in 0..32 {
+            let mut builder = Builder::new(&mut aligned.0[i..]);
+            builder.push("/init").unwrap();
```

Comment:
As the number of arguments also plays a role, one can add some variation here by
```rust
 builder.push("arg").unwrap();
```
in a variable number.